src/SprykerEco/Zed/Adyen/Business/Handler/Notification/AdyenNotificationHandler.php

```diff
@@ -93,10 +93,33 @@ protected function handleNotification(AdyenNotificationRequestItemTransfer $noti
             $this->utilEncodingService->encodeJson($notificationTransfer->getAdditionalData())
         );
 
+        if ($this->isDuplicatedAuthorisationNotification($notificationTransfer, $paymentAdyenOrderItems)) {
+            return;
+        }
+
         $this->writer->updatePaymentEntities(
             $statuses[$notificationTransfer->getEventCode()][$notificationTransfer->getSuccess()],
             $paymentAdyenOrderItems,
             $paymentAdyenTransfer
         );
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\AdyenNotificationRequestItemTransfer $notificationTransfer
+     * @param \Generated\Shared\Transfer\PaymentAdyenOrderItemTransfer[] $paymentAdyenOrderItems
+     *
+     * @return bool
+     */
+    private function isDuplicatedAuthorisationNotification(AdyenNotificationRequestItemTransfer $notificationTransfer, array $paymentAdyenOrderItems): bool
+    {
+        if ($notificationTransfer->getEventCode() === $this->config->getAdyenNotificationEventCodeAuthorisation()) {
```

Comment:
* Do an early return.
`if ($notificationTransfer->getEventCode() !== $this->config->getAdyenNotificationEventCodeAuthorisation()) {
return false
}`.